codersdk/deployment.go

```diff
@@ -1842,6 +1843,15 @@ Write out the current server config as YAML to stdout.`,
 			Group:       &deploymentGroupClient,
 			YAML:        "webTerminalRenderer",
 		},
+		{
+			Name:        "Allow Workspace Renames",
+			Description: "DEPRECATED: Allow users to rename their workspaces. Use only for temporary compatibility reasons.",
```

Comment:
Should we to mention here that it will be removed in a future release?

Reply:
Sounds good, added